src/docs/data/builders/tree.ts

```diff
@@ -28,43 +36,37 @@ const builder = builderSchema(BUILDER_NAME, {
 	],
 	states: [
 		{
-			name: 'collapsedGroups',
-			description: 'The list of IDs of items, who have a group that is not expanded.',
+			name: 'expanded',
+			description: 'The list of IDs of items, who have a group that is expanded.',
 		},
 		{
-			name: 'focusedItem',
-			description: 'The list item that is currently focused.',
+			name: 'selected',
+			description: 'The list of IDs of selected items.',
 		},
 		{
-			name: 'selectedItem',
-			description: 'The list item that is currently selected.',
+			name: 'firstSelected',
+			description: 'The ID of the first selected item, or `null` if none are selected.',
 		},
 	],
 	helpers: [
 		{
-			name: 'isCollapsedGroup',
+			name: 'isExpanded',
 			description:
-				'A derived store that returns a function that returns whether or not the item is collapsed.',
-			type: 'Readable<(itemId: string) => boolean>',
+				'A derived store that returns a function that returns whether or not the item is expanded.',
+			type: 'WithGet<Readable<(itemId: string) => boolean>>',
```

Comment:
Do we want to expose the `WithGet` to the consumer? We don't have this documented anywhere and it adds an element of WTF is this to someone reading the API ref.

It also would cause issues with other non-melt dependencies that may expect just a `Readable` type.

IMO we return the stores without the `get` methods and keep it simple.,

Reply:
We can keep it out of the documentation sure. I forgot about it being internal. But, having a get() method attached shouldn’t cause any issues. You can still pass it to functions that expect a Readable.

Reply:
Agree with removing it out of the documentation, but keeping the get method.